proxies/shared/resources/jsc/MessageBatches.Create.Validate.js

```diff
@@ -28,6 +28,28 @@ const isUndefined = (val) => {
     return typeof val === "undefined";
 }
 
+const validateNhsNumber = (nhsNumber, nhsNumberRegex) => {
+  if (!nhsNumberRegex.test(nhsNumber)) {
+    return false
+  }
+
+  const checkDigit = nhsNumber.substring(9);
```

Comment:
`const digits = nhsNumber.split('');`
`const checkDigit = digits[9];`

?

Reply:
Split apparently not supported with string separators (or blank separators) in RhinoJS